src/wkwebkit.cs

```diff
@@ -435,6 +443,10 @@ interface WKWebsiteDataStore : NSSecureCoding {
 	interface WKOpenPanelParameters	{
 		[Export ("allowsMultipleSelection")]
 		bool AllowsMultipleSelection { get; }
+
+		[Mac (10, 13, 4)]
```

Comment:
I don't recall if inheritance covers this but it's easier when reviewing if the `onlyOn64 : true` is added on members (even if the type has it).